### ISSUE TITLE
results_in_batches doesn't use credentials used to create saved search

### ISSUE THREAD

**Issue by artbq** [NetSweet/netsuite#372]
If I don't set NetSuite credentials with `NetSuite.configure` and instead pass them in the `search` call, they are not being used when I call `results_in_batches` on a search result.
```
NetSuite.configure do
  api_version '2017_1'
end
credentials = {:account=>"...", :consumer_key=>"...", :consumer_secret=>"...", :token_id=>"...", :token_secret=>"..."}
search = NetSuite::Records::InventoryItem.search({preferences: {page_size: 100}}, credentials)
search.total_pages # => 7
search.results_in_batches { |batch| puts batch.count } # => Savon::SOAPFault: (soapenv:Server.userException) Invalid account number.
```

**Comment by iloveitaly** [comment]
@artbq Ah, good catch! Could you submit a PR to fix this?

**Comment by iloveitaly** [comment]
Actually, looks like someone else recently submitted a PR for this! https://github.com/NetSweet/netsuite/pull/370

**Comment by iloveitaly** [comment]
This is fixed with 1d94a4898d350f6d6e8f6d1d10dce181098c3564

**Comment by artbq** [comment]
Sorry, haven't noticed the PR. Thanks for merging it!
